strings-csharp/ReplaceLineBreaksInAStringCSharp/ReplaceLineBreaksInAStringCSharp/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text.RegularExpressions;
+
+namespace ReplaceLineBreaksInAStringCSharp;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var stringReplaceResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheStringReplaceMethod();
+        Console.WriteLine(stringReplaceResult);
+        
+        var stringReplaceLineEndingsResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheStringReplaceLineEndingsMethod();
+        Console.WriteLine(stringReplaceLineEndingsResult);
+        
+        var regularExpressionReplaceResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheRegularExpressionReplaceMethod();
+        Console.WriteLine(regularExpressionReplaceResult);
+    }
+}
+
+public class ReplaceLineBrake
```

Comment:
This should be in its own file. Also, we can make it static which will clean up the Program.cs since we will not have to instantiate the class.